packages/plugin-core/package.json

```diff
@@ -125,6 +125,34 @@
                         "selectionExtract"
                     ]
                 },
+                "dendron.defaultTimestampDecorationFormat": {
+                    "default": "DATETIME_MED",
+                    "type": "string",
+                    "description": "Decide how human readable timestamp decoration is displayed",
+                    "enum": [
+                        "DATETIME_FULL",
+                        "DATETIME_FULL_WITH_SECONDS",
+                        "DATETIME_HUGE",
+                        "DATETIME_HUGE_WITH_SECONDS",
+                        "DATETIME_MED",
+                        "DATETIME_MED_WITH_SECONDS",
+                        "DATETIME_SHORT",
+                        "DATETIME_SHORT_WITH_SECONDS",
+                        "DATE_FULL",
+                        "DATE_HUGE",
+                        "DATE_MED",
+                        "DATE_MED_WITH_WEEKDAY",
```

Comment:
This is not exactly what was requested in #468 but close. 

weird, the type declaration from luxon doesn't include `DateTime.DATETIME_MED_WITH_WEEKDAY` (which is exactly what was requested).

Maybe this PR shouldn't be closing the issue.